app/controllers/posts_controller.rb

```diff
@@ -2,7 +2,24 @@
 
 class PostsController < ApplicationController
   def index
-    posts = Post.all
-    render status: :ok, json: { posts: }
+    posts = Post.order(created_at: :desc)
+    render_json({ posts: })
   end
+
+  def create
+    post = Post.new(post_params)
+    post.save!
```

Comment:
```suggestion
    Post.create!(post_params)
```

This is more concise.